src/main/java/mate/jdbc/model/Manufacturer.java

```diff
@@ -0,0 +1,64 @@
+package mate.jdbc.model;
+
+public class Manufacturer {
+    private long id;
+    private String name;
+    private String country;
+    private boolean isDeleted;
```

Comment:
В энтити классах всегда используем обертки, а не примитивы
```suggestion
    private Long id;
    private String name;
    private String country;
    private Boolean isDeleted;
```